modules/fledgeForGpt.js

```diff
@@ -31,12 +32,10 @@ export function init(cfg) {
   }
 }
 
-export function addComponentAuctionHook(next, bidRequest, componentAuctionConfig) {
+export function addComponentAuctionHook(next, adUnitCode, componentAuctionConfig) {
   const seller = componentAuctionConfig.seller;
-  const adUnitCode = bidRequest.adUnitCode;
   const gptSlot = getGptSlotForAdUnitCode(adUnitCode);
   if (gptSlot && gptSlot.setConfig) {
-    delete componentAuctionConfig.bidId;
```

Comment:
We deleted `bidId` because it was not in the schema, on the off chance the object might be checked against a schema for extra unapproved fields, given it is intended to be passed to sandboxed code.